lib/Doctrine/Common/Annotations/CachedReader.php

```diff
@@ -246,13 +246,17 @@ private function getLastModification(ReflectionClass $class)
         ));
     }
 
+    /**
+     * @param ReflectionClass $reflectionTrait
+     * @return int
+     */
     private function getTraitLastModificationTimes(ReflectionClass $reflectionTrait)
```

Comment:
The three dots don't work.

Compile Error: Cannot use positional argument after argument unpacking